web/src/hooks/useTaskLists.tsx

```diff
@@ -67,6 +67,7 @@ export default function useTaskLists(tasks: Task[]): UseTaskListsPayload {
   const [lists, setLists] = React.useState<TaskListDescription[]>(
     createLists(tasks)
   );
+  console.log("lists", lists)
```

Comment:
Question: Is this meant to be here? If so, I wonder if some descriptive text should be added to the output.